wrappers/python/simtk/openmm/app/gromacstopfile.py

```diff
@@ -840,18 +882,25 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
 
                 for fields in moleculeType.atoms:
                     params = self._atomTypes[fields[1]]
-                    
+
                     if len(fields) > 6:
                         q = float(fields[6])
                     else:
                         q = float(params[4])
 
-                    if self._defaults[1] == '1':
+                    if has_nbfix_terms:
+                        if self._defaults[1] != '2':
+                            raise NotImplemented
```

Comment:
Could you add a descriptive string to the exception so the user will know what aspect of their file isn't implemented?

Reply:
I didn't mean to add comments in the code.  I meant add a description to the exception so the user will know what the problem is.

```python
raise NotImplemented('Files with ... are not currently supported.')
```

Reply:
Thanks for the comment. I have updated the description.